uint8[] req
int32 cid
int32 round
---
bool resp